src/components/home/elements/BurnToken.vue

```diff
@@ -0,0 +1,750 @@
+                    functionIndex: BURN_FUNCIDX,
+                    functionData: transaction.prepareIssueAndBurn(BigNumber(this.amount).multipliedBy(this.tokenUnity)),
+                    signature: transaction.prepareExecContractSignature(this.contractId, BURN_FUNCIDX, transaction.prepareIssueAndBurn(BigNumber(this.amount).multipliedBy(this.tokenUnity)), this.attachment, BigNumber(CONTRACT_EXEC_FEE * VSYS_PRECISION), this.feeScale, BigNumber(this.timeStamp), this.getKeypair(this.addresses[this.address]).privateKey)
+                }
+                apiSchema = dataInfo
+            } else if (walletType === 'coldWallet') {
+                const coldDataInfo = {
+                    contractId: this.contractId,
+                    senderPublicKey: this.coldAddresses[this.address].publicKey,
+                    fee: CONTRACT_EXEC_FEE * VSYS_PRECISION,
+                    feeScale: FEE_SCALE,
+                    timestamp: this.dataObject.timestamp,
+                    functionIndex: BURN_FUNCIDX,
+                    functionData: transaction.prepareIssueAndBurn(BigNumber(this.coldAmount).multipliedBy(this.tokenUnity)),
+                    signature: this.coldSignature
+                }
+                apiSchema = coldDataInfo
+            }
+            const url = NODE_IP + '/contract/broadcast/execute'
+            this.$http.post(url, apiSchema).then(response => {
+                if (walletType === 'hotWallet') {
+                    this.pageId++
+                } else {
+                    this.coldPageId++
+                }
+            }, response => {
+                this.sendError = true
+            })
+        },
+        nextPage: function() {
+            this.pageId++
+            this.timeStamp = Date.now() * 1e6
+            this.hasConfirmed = false
+        },
+        coldNextPage: function() {
+            this.sendError = false
+            this.coldPageId++
+        },
+        prevPage: function() {
+            this.sendError = false
+            if (this.pageId === 1) {
+                this.$refs.sendModal.hide()
+            } else {
+                this.pageId--
+            }
+        },
+        coldPrevPage: function() {
+            this.sendError = false
+            if (this.coldPageId === 1) {
+                this.$refs.sendModal.hide()
+            } else {
+                this.coldPageId--
+            }
+        },
+        resetPage: function() {
+            this.amount = BigNumber(0)
+            this.pageId = 1
+            this.coldAmount = BigNumber(0)
+            this.coldPageId = 1
+            this.coldAddress = ''
+            this.scanShow = false
+            this.qrInit = false
+            this.paused = false
+            this.qrErrMsg = void 0
+            this.sendError = false
+            this.coldSignature = ''
+            this.coldAddress = this.walletType === 'coldWallet' ? this.selectedAddress : this.defaultColdAddress
```

Comment:
in issue token case, `this.coldAddress = ''` but? and why also init `this.coldAddress = ''` in line 478.